### ISSUE TITLE
Invalid latlng parameter

### ISSUE THREAD

**Issue by andyws** [chadly/Geocoding.net#81]
If I call ReverseGeocodeAsync( 51.441163, 0.000081) this generates a request:

```
GET https://maps.googleapis.com/maps/api/geocode/xml?latlng=51.441163,8.1E-05&sensor=false HTTP/1.1
```

which in turn generates:

```
<?xml version="1.0" encoding="UTF-8"?>
<GeocodeResponse>
 <status>INVALID_REQUEST</status>
 <error_message>Invalid request. Invalid 'latlng' parameter.</error_message>
</GeocodeResponse>

```

Suggestions?

**Comment by chadly** [comment]
See #74

**Comment by chadly** [comment]
Fixed via 42273c205ac3d7db935d35866744ae9f63c882f8
